### PR TITLE
Listing Page UI, Privacy Page, Terms & Conditions Page

### PR DESCRIPTION
**Pull request by philmillr**
- Added a little bit of UI styling for Listing page. I might add some more later
- Added the privacy page and linked it to the Listing page
- Added the Terms & Conditions Page and linked it to the Listing page

### CODE REVIEW

File: Update
Comment (kevinbae15) [inline Update:1]: wot is this

File: src/pages/Overview/Privacy/Privacy.jsx [hunk 1]
```diff
@@ -0,0 +1,495 @@
...
+class Privacy extends Component {
+/*
+  componentDidMount() {
+    document.title = 'Privacy Policy - TEN'
+    footerDelay()
+  }
+  componentDidUpdate() {
+    footerDelay()
+  }
+*/
```
Comment (kevinbae15) [inline src/pages/Overview/Privacy/Privacy.jsx]: remove any comments

File: src/pages/Overview/Privacy/Privacy.jsx [hunk 2]
```diff
@@ -0,0 +1,495 @@
...
+
+class PrivacyItem extends Component {
+  render() {
+    return (
+      <div className='PrivacyItem' style={{ float: this.props.float }}>
+        {this.props.text}
+      </div>
+    )
+  }
+}
```
Comment (kevinbae15) [inline src/pages/Overview/Privacy/Privacy.jsx]: seems like this isnt used anywhere

File: src/pages/Overview/Privacy/Privacy.jsx [hunk 3]
```diff
@@ -0,0 +1,495 @@
...
+    document.title = 'Privacy Policy - TEN'
+    footerDelay()
+  }
+  componentDidUpdate() {
+    footerDelay()
+  }
+*/
+  render() {
+    return (
+      <div className='everything'>
```
Comment (kevinbae15) [inline src/pages/Overview/Privacy/Privacy.jsx]: its a weird name for a class

File: src/pages/Overview/Privacy/Privacy.jsx [hunk 4]
```diff
@@ -0,0 +1,495 @@
...
+      </div>
+    )
+  }
+}
+
+function footerDelay() {
+  setTimeout(() => {
+    stickyFooter(0)
+  }, 1)
+}
```
Comment (kevinbae15) [inline src/pages/Overview/Privacy/Privacy.jsx]: can remove this

File: src/pages/Overview/Privacy/Privacy.jsx [hunk 5]
```diff
@@ -0,0 +1,495 @@
...
+  render() {
+    return (
+      <div className='everything'>
+        <div className='titleTC'>
+          <br />
+          Privacy Policy
+        </div>
+        <div className='lastupdated'>
+          <br />
+          Last updated November 08, 2019
```
Comment (kevinbae15) [inline src/pages/Overview/Privacy/Privacy.jsx:15]: damn, has this not been updated for that long?

File: src/pages/Overview/Terms/Terms.jsx [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+/*
+  componentDidMount() {
+    document.title = 'Terms and Conditions - TEN'
+    footerDelay()
+  }
+
+  componentDidUpdate() {
+    footerDelay()
+  }
+*/
```
Comment (kevinbae15) [inline src/pages/Overview/Terms/Terms.jsx]: same here with comments and unused components

File: src/pages/Overview/Terms/Terms.jsx [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+        <br />
+        <br />
+        <br />
+        <br />
+        <br />
+        <br />
+        <br />
+        <br />
+        <br />
+        <br />
```
Comment (kevinbae15) [inline src/pages/Overview/Terms/Terms.jsx]: i would change this out for some css

File: src/pages/Overview/Terms/Terms.jsx [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+  }
+}
+
+/*
+function footerDelay() {
+  setTimeout(() => {
+    stickyFooter(0)
+  }, 1)
+}
+*/
```
Comment (kevinbae15) [inline src/pages/Overview/Terms/Terms.jsx]: remove

File: updating merge conflict
Comment (kevinbae15) [inline updating merge conflict:1]: wot is this v2

File: src/pages/Sell/ListingPage.jsx [hunk 9]
```diff
@@ -59,42 +61,60 @@ export default function ListingPage() {
...
+              Submit
+            </button>
+
+        </form>
+
+      </Card.Body>
+    </Card>
+
+    <div className="w-100 text-center mt-2">
+    By continuing, you accept our <a href="/Privacy">privacy policy</a> and <a href="/Terms">terms and conditions</a>.
```
Comment (kevinbae15) [inline src/pages/Sell/ListingPage.jsx]:
```suggestion
    By continuing, you accept our <a href="/privacy">privacy policy</a> and <a href="/terms">terms and conditions</a>.
```

**Review by kevinbae15 — APPROVED** [review]
skimmed it and have some clean up comments

File: src/pages/Overview/Privacy/Privacy.jsx (hunk 1 above)
Comment (philmillr) [reply src/pages/Overview/Privacy/Privacy.jsx]: Will update in next PR! Updated locally!

File: src/pages/Overview/Privacy/Privacy.jsx (hunk 2 above)
Comment (philmillr) [reply src/pages/Overview/Privacy/Privacy.jsx]: Will update in next PR! Updated locally!

File: src/pages/Overview/Privacy/Privacy.jsx (hunk 3 above)
Comment (philmillr) [reply src/pages/Overview/Privacy/Privacy.jsx]: updated! Will see in next PR

File: src/pages/Overview/Terms/Terms.jsx (hunk 6 above)
Comment (philmillr) [reply src/pages/Overview/Terms/Terms.jsx]: updated for next PR!

File: src/pages/Overview/Terms/Terms.jsx (hunk 7 above)
Comment (philmillr) [reply src/pages/Overview/Terms/Terms.jsx]: done in next PR

**Review by HongjieWang98 — APPROVED** [review]

**Pull request merged**